ansible/roles/master/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+kubernetes_version: v1.0.3
+kubernetes_download_filename: "kubernetes-{{ kubernetes_version }}.tar.gz"
+kubernetes_download_url: "https://github.com/kubernetes/kubernetes/releases/download/{{ kubernetes_version }}/kubernetes.tar.gz"
+kubernetes_download_dir: /usr/local/src
```

Comment:
No, I don't have any preference toward `/usr` but I don't know which subdirectory in `/var` is more suitable. Maybe `/var/cache`?